values-observer-scs.yaml

```diff
@@ -421,89 +421,6 @@ dnation-kubernetes-monitoring:
           - port: "9180"
             interval: 30s
             path: /metrics
-## Uncomment only when SCS Zuul VM has been bootstrap with node and cadvisor exporters
```

Comment:
I see these sections also in values-observer, Should they also be deleted?

Reply:
IMO values-observer are generic, non SCS specific, so let the example of Zuul related values there